### PR TITLE
[JENKINS-62714] - Replace esclave with agent in French documentation

### PR DESCRIPTION
**Pull request by clywyk**
Replaced esclave with agent in French documentation.
_No other changes (grammatical or otherwise) have been made._

See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

### Proposed changelog entries

* Replace esclave with agent in French documentation
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/report-l10n.rb [hunk 1]
```diff
@@ -170,7 +170,7 @@ module ISO639
 dar|||Dargwa|dargwa
 day|||Land Dayak languages|dayak, langues
 del|||Delaware|delaware
-den|||Slave (Athapascan)|esclave (athapascan)
+den|||Slave (Athapascan)|agent (athapascan)
```
Comment (daniel-beck) [inline core/report-l10n.rb:173]: Nope.

https://en.wikipedia.org/wiki/Slavey_language

> Slavey (also Slave, Slavé) is an Athabaskan language

**Comment by daniel-beck** [comment]
>  No other changes (grammatical or otherwise) have been made.

Could you clarify whether that means the resulting translations may be newly ungrammatical? Or are you just saying you didn't fix what was wrong previously?

**Comment by clywyk** [comment]
> > No other changes (grammatical or otherwise) have been made.
> 
> Could you clarify whether that means the resulting translations may be newly ungrammatical? Or are you just saying you didn't fix what was wrong previously?

@daniel-beck I am working from a principle I'm learning at work that it's better to have a focused change. If I can, I'll go back through and look at the actual French, but I wanted this PR to only be the word change. The other changes I had made in two of the PRs I submitted yesterday had nothing to do with the esclave/agent change. It's not a question of "want" but of best practice.

**Comment by daniel-beck** [comment]
@clywyk That makes sense, thanks. We've had some problems in the past with bad replacements, so I was just making sure this doesn't introduce new problems.

File: core/report-l10n.rb (hunk 1 above)
Comment (clywyk) [reply core/report-l10n.rb]: @daniel-beck Removed the change, then did an interactive rebase. Could you resolve? Thanks.

**Review by markjacksonfishing — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by oleg-nenashev** [comment]
FTR I can read French well enough to approve this PR. Requesting a few additional reviews just in case

**Comment by oleg-nenashev** [comment]
Test error is caused by infra, should not be a blocker in this case.

```
[2020-06-14T14:06:01.628Z] [Pipeline] node

[2020-06-14T14:21:56.126Z] EC2 (aws) - Windows 2019 (i-05513133bfe940aeb) was marked offline: Connection was broken: java.io.EOFException

[2020-06-14T14:21:56.126Z] 	at java.base/java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2818)

[2020-06-14T14:21:56.126Z] 	at java.base/java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:3313)

[2020-06-14T14:21:56.126Z] 	at java.base/java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:907)

[2020-06-14T14:21:56.126Z] 	at java.base/java.io.ObjectInputStream.<init>(ObjectInputStream.java:350)

[2020-06-14T14:21:56.126Z] 	at hudson.remoting.ObjectInputStreamEx.<init>(ObjectInputStreamEx.java:49)

[2020-06-14T14:21:56.126Z] 	at hudson.remoting.Command.readFrom(Command.java:142)

[2020-06-14T14:21:56.126Z] 	at hudson.remoting.Command.readFrom(Command.java:128)

[2020-06-14T14:21:56.126Z] 	at hudson.remoting.AbstractSynchronousByteArrayCommandTransport.read(AbstractSynchronousByteArrayCommandTransport.java:35)

[2020-06-14T14:21:56.126Z] 	at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:63)

[2020-06-14T14:21:56.126Z] Caused: java.io.IOException: Unexpected termination of the channel

[2020-06-14T14:21:56.126Z] 	at hudson.remoting.SynchronousCommandTransport$ReaderThread.run(SynchronousCommandTransport.java:77)
```

**Review by alecharp — APPROVED** [review]

**Review by Vlatombe — APPROVED** [review]

**Comment by timja** [comment]
> EC2 (aws) - Windows 2019 (i-05513133bfe940aeb) was marked offline: Connection was broken: java.io.EOFException

looks like windows nodes may need to have there sizing reviewed as well, cc @slide. So far I've only touched the linux and linux high-mem.

**Comment by slide** [comment]
Hmmm, they are already at the same level as the current changes you made for Linux instances. I'll look at all the options.

**Comment by oleg-nenashev** [comment]
Question to @jenkinsci/core. Do we want to backport the change so that it gets shipped in 2.235.2 LTS? In general we do not backport localizations, but I suggest to make an exception for the obsolete agent terminology cleanup pull requests (there is another one from @daniel-beck I will need to handle then)

**Comment by jeffret-b** [comment]
I'm +1 on backporting this and other similar changes. Given the current worldwide focus on these sorts of issues, quick, easy wins like this are worthwhile. Particularly with the effort @clywyk has made to keep the changes focused.

**Comment by oleg-nenashev** [comment]
Created https://issues.jenkins-ci.org/browse/JENKINS-62714 as a backporting placeholder

**Comment by oleg-nenashev** [comment]
Thanks for your contribution @clywyk ! It should land in the next Jenkins weekly release

**Pull request merged**